README.md

```diff
@@ -1 +1,33 @@
 # rds-snapshot-sanitizer
+
+Create sanitized copy of RDS snapshots and share them with selected accounts.
+
+It works by restoring an unsanitized snapshot to a temporary cluster and executing sanitizing SQL queries against it, after which sanitized snapshot will be created and optionally shared with other accounts.
```

Comment:
I think it's better if you provide guide on how to run this locally (either with Docker or with Poetry)

Reply:
Also, I think it's useful to describe what IAM permissions are required for this

Reply:
> Also, I think it's useful to describe what IAM permissions are required for this

Yes, I'm thinking of creating a pod-identity terraform module for it int he style of https://github.com/terraform-aws-modules/terraform-aws-eks-pod-identity

> I think it's better if you provide guide on how to run this locally (either with Docker or with Poetry)

The thing is the tool needs to be connected to the RDS subnet for running the SQL query. I'll probably add a flag to set the postgres host to `localhost`, with the assumption that the user can connect their localhost to the RDS (via bastion port-forwarding for example).

Reply:
I see, if it's expected to run within AWS it's fine too, just add a note describing as such.